### DISCUSSION TITLE
Is there any chance that Skywalking ui will support scatter plots in the future

### DISCUSSION THREAD

**Discussion by bbcdabao** [apache/skywalking#11931] [General]
Is there any chance that Skywalking ui will support scatter plots in the future? It can intuitively discover the overall system performance distribution

**Comment by wu-sheng** [comment]
You means the graph? I can't see much difference from heatmap.

**Comment by bbcdabao** [comment]
![image](https://github.com/apache/skywalking/assets/45411947/ddf7067a-55d7-48e4-81a6-6dd29814dfcf)
It is this kind of scatter plot that can show the distribution of our BasicTraces

**Reply by wu-sheng** [reply]
If you like this, choose Pinpoint, I know this graph is from it.
We had provided heatmap, but not that useful, if you like, you could add it back.

**Reply by bbcdabao** [reply]
ok That's what I meant， It will be a UI plugin that already has some effects
![image](https://github.com/apache/skywalking/assets/45411947/71d03de7-9934-4c57-9ab0-f158644ac58d)
This is not PinPoint but similar

**Reply by wu-sheng** [reply]
SkyWalking already have that, but not in this way.

https://skywalking.apache.org/blog/boost-root-cause-analysis-quickly-with-skywalking-new-trace-metrics-association-feature/

This blog provides a lot of details about the practice of finding traces from metrics

**Reply by bbcdabao** [reply]
ok thank you

**Comment by bbcdabao** [comment]
When inspecting the system or locating problems, you can see some points to choose to enter the Trace Tree

**Reply by wu-sheng** [reply]
That is what p99 and metrics/trace association designed for.
Long tail is a common case, but the dots are not that useful.

**Comment by bbcdabao** [comment]
I hope it can be submitted to Skywalking, but the code needs to be tidied up

**Reply by wu-sheng** [reply]
There is not post-aggregation in skywalking, that is why we asked you to choose Pinpoint, which is hbase based, and good at running analysis.

**Reply by bbcdabao** [reply]
Thank you for your reply
